python/dask_cudf/dask_cudf/core.py

```diff
@@ -34,19 +32,6 @@
 DASK_VERSION = LooseVersion(dask.__version__)
 
 
-def optimize(dsk, keys, **kwargs):
-    flatkeys = list(flatten(keys)) if isinstance(keys, list) else [keys]
-    dsk, dependencies = cull(dsk, flatkeys)
-    dsk, dependencies = fuse(
-        dsk,
-        keys,
-        dependencies=dependencies,
-        ave_width=_globals.get("fuse_ave_width", 1),
```

Comment:
Was wondering if we should be setting this somehow to ensure it is `1`. However that appears to [be the default anyways]( https://github.com/dask/dask/blob/5bda9c4fed4b9f1bfcd94a187d509bf8d77ce877/dask/dask.yaml#L15 ). So that doesn't seem needed

The other `cull`ing steps happen naturally as part of `optimize` anyways so no need to reproduce them otherwise either